sql-project-1/sql_project.sql

```diff
@@ -0,0 +1,97 @@
+CREATE TABLE customer_Aj
+(customer_id smallint,
+name varchar(20),
+location varchar(20),
+total_expenditure varchar(20));
+
+INSERT INTO customer_Aj (customer_id, name, location, total_expenditure)
+VALUES (1701, "John", 'Newport Beach, CA', '2000'),
+(1707, 'Tracy', 'Irvine, CA', '1500'),
+(1711, 'Daniel', 'Newport Beach, CA', '2500'),
+(1703, 'Ella', 'Santa Ana, CA', '1800'),
+(1708, 'Mel', 'Orange, CA', '1700'),
+(1716, 'Steve', 'Irvine, CA', '18000');
+
+UPDATE customer_Aj
+SET total_expenditure = '1800'
+WHERE customer_id IN (1716);
+
+ALTER TABLE customer_Aj 
+ADD gender varchar(20);
+
+UPDATE customer_Aj 
+SET gender = 'M'
+WHERE customer_id IN (1701, 1711, 1716);
+
+UPDATE customer_Aj 
+SET gender = 'F'
+WHERE customer_id IN (1707, 1703, 1708);
+
+DELETE FROM customer_Aj 
+WHERE customer_id=1716;
+
+ALTER TABLE customer_Aj 
+ADD store varchar(20);
+
+ALTER TABLE customer_Aj 
+DROP COLUMN store;
+
+SELECT * FROM customer_Aj;
+
+SELECT name n, total_expenditure total_exp
+FROM customer_Aj;
+
+ALTER TABLE customer_Aj 
```

Comment:
This is better achieved by using a MODIFY statement instead of deleting and re-adding the column